instrumentation-genai/opentelemetry-instrumentation-vertexai-v2/example/main.py

```diff
@@ -0,0 +1,16 @@
+import vertexai
+from vertexai.generative_models import GenerativeModel
+
+vertexai.init(location="us-central1")
```

Comment:
can this be set via ENV in the normal client? It would be nicer to adjust .env vs edit the source

Reply:
I've removed it altogether since the SDK will read it from gcloud config. Incorporated into https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3123